datafusion/core/src/physical_planner.rs

```diff
@@ -246,6 +246,7 @@ fn create_physical_name(e: &Expr, is_first_expr: bool) -> Result<String> {
             args,
             filter,
             order_by,
+            ..
```

Comment:
As as style thing I have found it helpful to not use a `..` match all, and instead explicitly ignore each field like

```suggestion
            null_treatment: _,
```

With this treatment, if/when someone adds a new field to `AggregateFunction` in the future, the compiler will tell them all the places that may need to be changed.

Reply:
Fixed. Thanks for the suggestion!